core/experiments/manager.php

```diff
@@ -785,6 +790,18 @@ private function should_show_hidden() {
 		return defined( 'ELEMENTOR_SHOW_HIDDEN_EXPERIMENTS' ) && ELEMENTOR_SHOW_HIDDEN_EXPERIMENTS;
 	}
 
+	private function create_dependency_class( $dependency_name, $dependency_args ) {
+		if ( class_exists( $dependency_name ) ) {
+			return $dependency_name::instance();
+		}
+
+		if ( ! empty( $dependency_args ) ) {
+			return new Wrap_Core_Dependency( $dependency_args );
+		}
```

Comment:
What the priority should be? maybe we first want the Wrap and then the instance?
IDK